_source/_posts/2024-04-30-express-universal-logout.md

```diff
@@ -635,8 +701,8 @@ export const Todos = () => {
         });
 
         if (!res.ok){if (res.status === 401) {
-// Redirect user back to the sign in page
-window.location.href = '/';
+        // Redirect user back to the sign in page
+        window.location.href = '/';
```

Comment:
out of curiosity, why are using window API here to redirect to home. I thought this is handled already inside the todo app, can't we use React Router?

Reply:
will pick up on this in next PR